pkg/pos/prove_test.go

```
@@ -0,0 +1,51 @@
+
+	key := make([]byte, utils.KeyLen)
+	if _, err := rand.Read(key); err != nil {
+		t.Fatal(err)
+	}
+
+	// 1. Plot
+	_, err := PlotDisk(plotPath, fsType, k, 100*1024*1024, key, false)
+	if err != nil {
+		t.Fatalf("cannot plot: %v", err)
+	}
+	defer os.Remove(plotPath)
```

golangci-lint (2.12.2)

[error] Error return value of `os.Remove` is not checked

(errcheck)